subscription/src/main/java/org/killbill/billing/subscription/api/user/DefaultSubscriptionBase.java

```diff
@@ -687,6 +690,30 @@ public List<SubscriptionBillingEvent> getSubscriptionBillingEvents(final Version
         }
     }
 
+
+    private DateTime alignToNextBCD(final Plan curPlan, final PlanPhase curPlanPhase, final DateTime originalDate, final SubscriptionCatalog catalog, final Integer bcdLocal, final InternalTenantContext context) throws SubscriptionBaseApiException, CatalogApiException {
```

Comment:
`bcdLocal` -> `catalogChangeEffectiveDate`

Reply:
What about [this](https://github.com/killbill/killbill/pull/1663/files#diff-6c9485b2174859a0d155b69e387e4b868e364c2608ba08fd0a2db9c49101fc46R645) 🤔 ?

Reply:
Are you suggesting I change `originalDate` -> `catalogChangeEffectiveDate` or `bcdLocal` -> `catalogChangeEffectiveDate` ?

Reply:
`bcdLocal` -> `catalogChangeEffectiveDate` -- it's not a _bcd_? 🤔

Reply:
humm - scratch that. I'm wondering if I read an earlier version of the code (see next comment).